Source/AwsCommonRuntimeKit/io/Stream.swift

```diff
@@ -4,106 +4,99 @@
 import AwsCIo
 import Foundation
 
-// swiftlint:disable trailing_whitespace
+/// This is for streaming input.
+public protocol IStreamable {
```

Comment:
Can we improve the naming here? What does the `I` stand for? 
Would just `Stream` or `InputStream` be a better fit? Its ok if the name doesn't end in `--able`, the Swift standard library has many protocols that don't follow that naming convention such as `Sequence` and `Collection`.

Reply:
Unfortunately, Swift already has Stream & InputStream classes. I stands for Input.